static/js/browserDetect.js

```diff
@@ -0,0 +1,19 @@
+var isChrome = !!window.chrome && !!window.chrome.webstore;
+var isEdge = !!window.StyleMedia;
+var isFirefox = typeof InstallTrigger !== 'undefined';
+var isOpera = (!!window.opr && !!opr.addons) || !!window.opera || navigator.userAgent.indexOf(' OPR/') != -1;
+var isSafari = navigator.userAgent.indexOf("Safari") > -1;
+
+if (isChrome) {
+  document.getElementById("chromebrowser").classList.add('activeBrowser');
```

Comment:
`.getElementById` -> `.querySelector` for each one, and maybe target the `.browser-xxx`s? Would also mean the meta in each `<img>` won't need to be specified